Documentation/CodeSnippets/TypoScriptSyntax/CodeBlockInvalidCondition.rst.txt

```diff
@@ -0,0 +1,12 @@
+.. Automatic screenshot: Remove this line if you want to manually change this file
+
+.. code-block:: typoscript
+   :caption: Extension examples, file Configuration/TypoScript/Syntax/CodeBlockInvalidCondition/setup.typoscript
+
+   myIdentifier = TEXT
+   myIdentifier {
+      value = foo
+      [frontend.user.isLoggedIn]
+         value = bar
+      [end]
```

Comment:
Just to make sure that conditions are possible inside code blocks now, I ask for confirmation